### PR TITLE
030 routes

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by mastemm — APPROVED** [review]
looks ok

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
     <div className="App">
-      <HomePage />
       <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/marvel" element={<Marvel />} />
+        <Route path="/starwars" element={<Starwars />} />
+        <Route path="/manga" element={<Manga />} />
+        <Route path="/dccomics" element={<Dccomics />} />
+        <Route path="/autresheros" element={<Autresheros />} />
+
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:23]: Blank Line

File: frontend/src/components/HomePage/HomePageHeader/index.jsx [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+            <li className="manga"> </li>
+          </Link>
+          <Link to="/dccomics">
+            <li className="dccomics"> </li>
+          </Link>
+          <Link to="/autresheros">
+            <li className="autresheros"> </li>
+          </Link>
+        </ul>
+        {/*  */}
```
Comment (LuigiMenez) [inline frontend/src/components/HomePage/HomePageHeader/index.jsx:25]: Dead code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Remove a dead code HomepageHeader .jsx and blank line23 App.jsx

**Pull request merged**